### PR TITLE
replace the term docker with container

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@gabemontero ptal, request from rcm.  unfortunately going to need to be backported as well.

**Comment by gabemontero** [comment]
/lgtm

but the error in the failed test run is an odd one ... it is during the docker build, though I don't see how your change could have induced it

**Comment by bparees** [comment]
indeed that is weird... let's try a retest and then i'll investigate further

/retest

File: 2/Dockerfile.rhel7 [hunk 1]
```diff
@@ -27,11 +27,10 @@ LABEL io.k8s.description="Jenkins is a continuous integration server" \
       io.openshift.s2i.scripts-url=image:///usr/libexec/s2i
 
 # Labels consumed by Red Hat build service
-LABEL com.redhat.component="openshift-jenkins-2-docker" \
+LABEL com.redhat.component="openshift-jenkins-2-container" \
       name="openshift3/jenkins-2-rhel7" \
       version="3.11" \
       architecture="x86_64" \
-      release="7"
```
Comment (bparees) [inline 2/Dockerfile.rhel7]: @gabemontero it was this. i need to remove the trailing slash above.

Comment (bparees) [reply 2/Dockerfile.rhel7]: weird error/failure mode though.

**Pull request merged**